### ISSUE TITLE
Comics with the word "Special" in their filename get improper name in Kavita collection.

### ISSUE THREAD

**Issue by apoapostolov** [Kareadita/Kavita#1379]
The file named: E:\/OneDrive/Comics/~WEEKLY\2022.06.29 Weekly Pack\2022.06.29 INDIE Week\Transformers - Fate of Cybertron Special (2022).cbr

How it appears in Kavita:

![image](https://user-images.githubusercontent.com/4083812/179391693-4c705817-c227-4d25-9ed2-2ba50db1ddb8.png)

What happens is that when a comic contains the name "Special" it will assume the name of its parent directory as the name of the series, even if the parent folder has nothing to do with the series. This seems to be related to the logic that searches for the series that need to be assigned the comic as a Special issue. The logic seems flawed when the comics aren't organized in that manner. 

Probably future scraping of metadata will help but for now, I would appreciate it if the Special logic can be turned off via a setting.

**Comment by majora2007** [comment]
Sorry, I have no plans to introduce a setting to toggle different ways to parse files. That opens the door to maintenance and testing I don't want to have to support.

**Comment by therobbiedavis** [comment]
Sorry to comment on something closed, but is this parsing the actual intended behavior? I thought it would try to grab the series name from the filename instead of the base folder. This kind of seems like a parser bug.

**Comment by majora2007** [comment]
@therobbiedavis Specials can't grab the series name from the file usually due to them being named some random title of the special, hence why it falls back to the folder. This is based on experience with other users naming conventions.

If we just assume the title is the special, it's going to be a separate series as we can't accurately parse just a series on its own without volume or chapter markers.

**Comment by therobbiedavis** [comment]
Fair enough. I need to double check the wiki to see if this behavior is documented.
